src/store/sagas.js

```diff
@@ -0,0 +1,46 @@
+import { call, put, takeLatest } from 'redux-saga/effects'
+import giphy from '../api/giphy'
+
+function* fetchTrending (action) {
```

Comment:
Do you notice the pattern on all of your sagas? You could abstract that pattern to avoid code repetition. `createSaga(callFunc, callArgs, successAction, failedAction)`